FabricExample/package.json

```diff
@@ -5,46 +5,40 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "ts-check": "yarn tsc --noEmit",
-    "lint-check": "yarn eslint src/ && yarn prettier --check src/",
-    "postinstall": "patch-package"
+    "test": "jest"
   },
   "dependencies": {
-    "@react-native/babel-plugin-codegen": "^0.0.6",
-    "@react-navigation/native": "^6.0.12",
-    "@react-navigation/native-stack": "^6.8.0",
-    "patch-package": "^6.4.7",
+    "patch-package": "^6.5.0",
     "postinstall-postinstall": "^2.1.0",
-    "react": "18.1.0",
-    "react-native": "0.70.3",
-    "react-native-gesture-handler": "link:../",
-    "react-native-safe-area-context": "4.4.1",
-    "react-native-screens": "3.18.2"
+    "react": "18.2.0",
+    "react-native": "0.71.0-rc.5",
+    "react-native-gesture-handler": "link:../"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
-    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-env": "^7.14.0",
     "@babel/runtime": "^7.12.5",
-    "@react-native-community/eslint-config": "^2.0.0",
-    "babel-jest": "^26.6.3",
-    "eslint": "^7.15.0",
-    "jest": "^26.6.3",
-    "metro-react-native-babel-preset": "0.72.3",
-    "mkdirp": "^1.0.4",
-    "prettier": "^2.2.1",
-    "react-test-renderer": "^18.1.0",
-    "typescript": "^4.5.5"
-  },
-  "resolutions": {
-    "@typescript-eslint/eslint-plugin": "^5.17.0",
-    "@typescript-eslint/parser": "^5.17.0",
-    "eslint-plugin-react-native": "^4.0.0",
-    "@types/react": "18.0.0"
+    "@react-native-community/eslint-config": "^3.0.0",
+    "@tsconfig/react-native": "^2.0.2",
+    "@types/jest": "^29.2.1",
+    "@types/react": "^18.0.24",
+    "@types/react-test-renderer": "^18.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
+    "babel-jest": "^29.2.1",
+    "eslint": "^8.19.0",
+    "jest": "^29.2.1",
+    "metro-react-native-babel-preset": "0.73.5",
+    "prettier": "^2.4.1",
+    "react-test-renderer": "18.2.0",
+    "typescript": "4.8.4"
   },
   "jest": {
     "preset": "react-native"
+  },
+  "resolutions": {
+    "@react-native-community/cli-platform-android": "10.0.0-alpha.4"
```

Comment:
no longer necessary, don't forget to update yarn.lock

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/2313/commits/f6df763638b1b41cd2aaa1cf08d2b396e55676e3.